docs-md/apireference/c.md

```diff
@@ -5270,6 +5315,70 @@ mtev_time_toggle_tsc(mtev_boolean enable)
 
 ### U
 
+#### mtev_url_decode
+
+>Decode a url encoded input buffer into the provided output buffer.
+
+```c
+int
+mtev_url_decode(const char *src, size_t src_len, unsigned char *dest, size_t dest_len)
+```
+
+
+  * `src` The buffer containing the encoded content.
+  * `src_len` The size (in bytes) of the encoded data.
+  * `dest` The destination buffer to which the function will produce.
+  * `dest_len` The size of the destination buffer.
+  * **RETURN** The size of the decoded output.  Returns zero is dest_len is too small.
+
+mtev_url_decode decodes input until an the entire input is consumed or until an invalid url-encoded character is encountered.
```

Comment:
The current description does not explain what will happen in regards to the return code if this situation occurs.  Will I still get back 0 or a different return code?